compiler/dyno/include/chpl/util/filesystem.h

```diff
@@ -65,6 +65,28 @@ std::error_code makeTempDir(std::string dirPrefix, std::string& tmpDirPathOut);
  */
 std::error_code deleteDir(std::string dirname);
 
+/*
+ * Returns the current working directory. Does not report failures. Use
+ * currentWorkingDir() if you need error reports.
+ */
+std::string getCwd();
```

Comment:
I think this particular variation should just live in `chpldoc` (it is not powerful/general enough).